dev_readme.md

```diff
@@ -1,30 +1,78 @@
+  Note: ***DO NOT UPLOAD THESE .ENV FILES ONTO GITHUB*** It's the equivalent of posting your passport ID on Twitter. Don't do it.
+
+### Open Humans Project Creation
+This step is a little tricky, but you're almost there. 
+
+1. Login to [OpenHumans](openhumans.org). 
+2. Navigate to [Manage Project](https://www.openhumans.org/direct-sharing/projects/manage/).
+3. Create a new Oauth2 Data Request Project on Open Humans. 
+4. Populate the required fields as directed. 
+5. In the `Redirect URL` field, enter `http://localhost:8000/openhumans/complete`.
+6. Click `Create Project`.
+
+You'll need to obtain the Client ID and Client Secret for the OAuth project you've created. 
+
+1. Navigate to [Manage Project](https://www.openhumans.org/direct-sharing/projects/manage/).
+2. Click the newly created project. 
+3. Copy and paste somewhere safe the `Client ID` and `Client Secret` under `OAuth2 Credentials`. 
+
+### Project Configuration 
+Within `.env` there are four different fields that must be populated for the project to build.
+
+#### Django Secret Key Generation
+1. Open `.env` in your favorite editor, and using `python3`, follow the instructions under the `django` header to copy/paste the ggenerated secret key to `DJANGO_SECRET_KEY` 
 
-***DO NOT UPLOAD THESE .ENV FILES ONTO GITHUB*** It's the equivalent of posting your passport ID on Twitter. Don't do it.
+The following...
+`DJANGO_SECRET_KEY=__CHANGEME__`
 
-#### 4. Open up Terminal or Command Line in the root level of the repository and type in the following command: `docker-compose up`
+should look something like this, where XXXXXXXXXXXXXXXXXXXXXX is the generated key. 
+`DJANGO_SECRET_KEY=XXXXXXXXXXXXXXXXXXXXXX`
 
-#### 5. Open your browser to `http://localhost:8000`
+#### Configuring OpenHumans
+Further down the `.env` file, you should see three texts: `OPENHUMANS_CLIENT_ID`, `OPENHUMANS_CLIENT_SECRET`, and `OPENHUMANS_APP_BASE_URL`.
 
-*This will display the AutSPACEs website.*
+1. Copy the `Client ID` and `Client Secret` that you've copied previously into `OPENHUMANS_CLIENT_ID` and `OPENHUMANS_CLIENT_SECRET`, respectively.
+It should look something like this, where XXXXXXXXXXXXXXXXXXXXXX are the inputted information. 
+```
+OPENHUMANS_CLIENT_ID=XXXXXXXXXXXXXXXXXXXXXX
+OPENHUMANS_CLIENT_SECRET=XXXXXXXXXXXXXXXXXXXXXX
+````
+
+2. Type the following url into `OPENHUMANS_APP_BASE_URL`.
+It should look like this!
+
+```
+OPENHUMANS_APP_BASE_URL="http://localhost:8000"
+```
+
+3. You're done!
+
+## Running the Project
+The project builds and runs in a self-contained environment called Docker. This means developer environments are preserved, and you don't need to mess with any dependency management or installation.
+
+1. Open up Terminal or Command Line in the root level of the repository and execute the following command: `./docker-run.sh`
+2. Open your browser to `http://localhost:8000`
+
+*This will display the AutSPACEs website.* Yay!
```

Comment:
These are really nice instructions! I think it's worth mentioning at the end here that once you get the Autspaces display you should not necessarily expect a full functional site. Just so that users know when they click through some pages and end up at a NoReverseMatch error, that's not a mistake on their end

Reply:
Thank you! 

Makes sense. Added! Let me know if it works for you!